packages/node_modules/@node-red/editor-client/src/js/ui/userSettings.js

```diff
@@ -101,10 +101,25 @@ RED.userSettings = (function() {
         RED.tray.show(trayOptions);
     }
 
+    function locale2Name(lc) {
```

Comment:
Please do not use `2` as an abbreviation for `To` - that is not the style we use in this code.

Reply:
changed `locale2Name` to `localeToName`.